src/analysis.rs

```diff
@@ -276,6 +276,9 @@ pub struct ModuleInfo {
   /// or `@import { SomeType } from "npm:some-module"`.
   #[serde(skip_serializing_if = "Vec::is_empty", default)]
   pub jsdoc_imports: Vec<JsDocImportInfo>,
+  /// Source map URL extracted from sourceMappingURL comment
+  #[serde(skip_serializing_if = "Option::is_none", default)]
+  pub source_map_url: Option<SpecifierWithRange>,
```

Comment:
Can you add a serialization/deserialization test for this? We need to ensure it's somewhat stable.